README.md

```diff
@@ -1 +1,81 @@
-# angular-with-go
\ No newline at end of file
+# Bold BI Embedded Sample in Angular with Go
+
+This Bold BI Angular with Go sample contains the Dashboard embedding sample. In this sample, the Angular application acts as the front-end, and the Go sample act as the back-end application. This sample demonstrates the dashboard rendering with the available dashboard in your Bold BI server.
+
+## Dashboard view
+
+![Dashboard View](https://github.com/boldbi/aspnet-core-sample/assets/91586758/966e2735-e9ee-469a-9781-4e4f23021b04)
+
+ ## Requirements/Prerequisites
+ 
+ * [Go installer](https://go.dev/dl/)
+ * [Node.js](https://nodejs.org/en/)
+ * [Visual Studio Code](https://code.visualstudio.com/download)
+ 
+ > **NOTE:** Node.js v14.16 to v18.18 are supported.
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox.
+
+ ## Configuration
+
+  * Please get the [Angular with Go] (https://github.com/boldbi/angular-with-go-sample) sample from GitHub.
```

Comment:
Remove space between Text and link.
![Uploading image.png…]()

Reply:
Resolved.